### PR TITLE
Restore mouse button release hack in title widget event filter

### PR DESCRIPTION
**Pull request by ilya-fedin**

### CODE REVIEW

File: Telegram/SourceFiles/window/window_title_qt.cpp [hunk 1]
```diff
@@ -255,6 +255,10 @@ bool TitleWidgetQt::eventFilter(QObject *obj, QEvent *e) {
 
 			if (e->type() == QEvent::MouseMove
 				&& mouseEvent->buttons() == Qt::NoButton) {
+				if (_mousePressed) {
```
Comment (mymedia2) [inline Telegram/SourceFiles/window/window_title_qt.cpp:258]: Redundant `if`. You could simply assign the desired value to the boolean variable without a pre-check.

Comment (ilya-fedin) [reply Telegram/SourceFiles/window/window_title_qt.cpp:258]: it is already merged

Comment (mymedia2) [reply Telegram/SourceFiles/window/window_title_qt.cpp:258]: That doesn't matter. This `if` remains pointless.

Comment (ilya-fedin) [reply Telegram/SourceFiles/window/window_title_qt.cpp:258]: image

**Pull request merged**